protobufs/livekit_metrics.proto

```diff
@@ -23,6 +23,20 @@ enum MetricLabel {
   AGENTS_LLM_TTFT = 0; // time to first token from LLM
   AGENTS_STT_TTFT = 1; // time to final transcription
   AGENTS_TTS_TTFB = 2; // time to first byte
+  CLIENT_FREEZE_COUNT = 3;                            // Number of video freezes
```

Comment:
Can we make these a little longer? Sorry, would be good to have these in the name I think.
Something like
`CLIENT_VIDEO_SUBSCRIBER_<property>`
include the media type and direction in the name.